### PR TITLE
TemplateImports cleanup

### PR DESCRIPTION
**Pull request by benmccann**
I noticed a couple minor code issues while working on the Gradle integration

### CODE REVIEW

File: framework/src/build-link/src/main/java/play/TemplateImports.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
     ));
 
   static {
-    List<String> minimalJavaImports = new ArrayList();
+    List<String> minimalJavaImports = new ArrayList<String>();
```
Comment (PromanSEW) [inline framework/src/build-link/src/main/java/play/TemplateImports.java]: You can use dianond operator because it is Java 7+ code

Comment (benmccann) [reply framework/src/build-link/src/main/java/play/TemplateImports.java]: I don't think the build-link project is. Here's what I get if I try:

```
[error] playframework/framework/src/build-link/src/main/java/play/TemplateImports.java:28: diamond operator is not supported in -source 1.6
[error]   (use -source 7 or higher to enable diamond operator)
[error]     List<String> minimalJavaImports = new ArrayList<>();
```

Comment (PromanSEW) [reply framework/src/build-link/src/main/java/play/TemplateImports.java]: Got it

**Comment by marcospereira** [comment]
Thank you, @benmccann. Just our curiosity, in what part of Gradle integration are you currently working?

Best.

**Comment by benmccann** [comment]
Just rebasing some old PRs I opened a long time ago

**Pull request merged**